crates/packager/src/config/mod.rs

```diff
@@ -488,6 +509,84 @@ impl AppImageConfig {
     }
 }
 
+/// The Linux pacman configuration.
+#[derive(Clone, Debug, Default, Deserialize, Serialize)]
+#[cfg_attr(feature = "schema", derive(schemars::JsonSchema))]
+#[serde(rename_all = "camelCase", deny_unknown_fields)]
+#[non_exhaustive]
+pub struct PacmanConfig {
+    // Arch Linux Specific settings.
+    /// List of Pacman dependencies.
+    pub depends: Option<Vec<String>>,
+    /// Additional packages that are provided by this app.
+    pub provides: Option<Vec<String>>,
```

Comment:
Should sidecars (external binaries) be included in this or rather what does this option do?

Reply:
No, sidecars need not be included, because they are already included in the package.

They are separate dependencies, and will be installed by `pacman` . It must be filled by user itself.